pkg/model/model.go

```diff
@@ -214,86 +206,3 @@ func readGob(filePath string, object interface{}) error {
 	}
 	return nil
 }
-
```

Comment:
How about comment this PR to explain how came these functions and why we could remove them?

Reply:
Done